### DISCUSSION TITLE
.theia to be or not to be ?

### DISCUSSION THREAD

**Discussion by Niklas-Dahlquist** [eclipse-theia/theia#10032] [Ideas] [answered]
Hi,

Since the introduction of support for VS-Code extensions the need for a .vscode folder was introduced. What I understand this has made .theia folder obsolete?

Is it time to remove it?

/Niklas

**Comment by maheshrajrp** [comment]
As far I have worked with, .theia folder is still used to store preferences of Theia.

**Comment by vince-fugnitto** [comment, marked as answer]
@Niklas-Dahlquist the support of `.theia` (and `.vscode` as a fallback) predates support for vscode extensions. We want to give applications the ability to define their own configuration folder (they can name it however they like and `.theia` is the default), so I do not believe we want to remove it at this time and it would be a major breaking change for apps and users.
